test/components/dashboard/multiple-choice-answer_spec.js

```diff
@@ -48,4 +59,75 @@ describe('<MultipleChoiceAnswer />', () => {
       expect(wrapper.find(Choice).at(1).prop('selected')).to.equal(false)
     })
   })
+
+  describe('when there is no choice with isCorrect property equal to `true`', () => {
+    it('should pass `correctAnswerDefined=false` prop to choices', () => {
```

Comment:
It seems like the logic worth testing here is that `correctAnswerDefined` is computed properly, more so than that it's passed correctly as a prop. I'd consider just testing that function on its own - that test might be less likely to break in the future.

Reply:
Yeap. But on the other hand, you test less funcionality. I'm usually trying to cover more code and functionality as long as it's easy.